prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -145,6 +145,12 @@ fi_ibv_rdm_tagged_prepare_conn_memory(struct fi_ibv_rdm_ep *ep,
 				(void **) &conn->rbuf_mem_reg, size);
 	assert(conn->r_mr);
 
+	conn->ack_mr = ibv_reg_mr(ep->domain->pd, &conn->sbuf_ack_status,
+		sizeof(conn->sbuf_ack_status),
+		IBV_ACCESS_LOCAL_WRITE | IBV_ACCESS_REMOTE_WRITE);
+
+	assert(conn->ack_mr);
```

Comment:
I looked on required sub-sequences to handle this error. I would prefer to make it in separated PR as error handling in connection management. There is a quite amount of changes not related with subject of this PR.